src/Mono.Android/Mono.Android.targets

```diff
@@ -128,6 +129,7 @@
         SourceFiles="$(IntermediateOutputPath)__dex\classes.dex"
         DestinationFiles="$(OutputPath)mono.android.dex"
     />
+    <Touch Files="$(OutputPath)mono.android.dex" />
```

Comment:
@atsushieno: Something is very fishy with that `grep`, as it's not finding the target definitions within the repo:

```
$ git grep GenerateMonoAndroidDex
src/Mono.Android/Mono.Android.targets:  <Target Name="_GenerateMonoAndroidDex16"
src/Mono.Android/Mono.Android.targets:  <Target Name="_GenerateMonoAndroidDex18"
```